tests/external_tests/person/test_person.py

```diff
@@ -23,6 +23,21 @@ def test_player_instance_position_arguments(self):
         self.assertEqual(self.player.fullname, "Ty France")
         self.assertEqual(self.player.link, "/api/v1/people/664034")
 
+    def test_get_persons(self):
+        # set draft id
+        player_ids = '605151,592450'
+
+        # call get_persons return list of players objects
+        players = self.mlb.get_persons('605151,592450')
```

Comment:
Can you make this accept a list of id's? instead of a single string or both? the method join might help

Reply:
That way you can use get_people_id well with it

Reply:
Should have added support